neo4j/_async/driver.py

```diff
@@ -62,11 +64,16 @@
     URI_SCHEME_NEO4J_SECURE,
     URI_SCHEME_NEO4J_SELF_SIGNED_CERTIFICATE,
 )
+from .bookmark_manager import (
+    AsyncNeo4jBookmarkManager,
+    T_BmConsumer as _T_BmConsumer,
+    T_BmSupplier as _T_BmSupplier,
+)
 from .work import AsyncSession
 
 
 class AsyncGraphDatabase:
-    """Accessor for :class:`neo4j.Driver` construction.
+    """Accessor for :class:`neo4j.AsyncDriver` construction.
     """
 
     if t.TYPE_CHECKING:
```

Comment:
oh, you have to explicitly check for this?

Reply:
You *can* make the distinction, you don't have to. But since the driver already had these config classes about that parse kwarg dicts, I figured the least intrusive thing to do would be to only explicitly state the available config options to the type checker (most IDEs should also be able to pick this up for auto completion), but continue using `**kwargs` and the config classes at runtime.